index.js

```diff
@@ -106,10 +106,19 @@ module.exports = {
       'components/dialog/dialog-theme.scss'
     ];
 
-    var pathBase = this.project.nodeModulesPath;
-    var angularMaterialPath =  'angular-material-source/src';
-
-    var angularScssFiles = new Funnel(path.join(pathBase,angularMaterialPath), {
+    /*
+      Find the angular-material-source module in a manner that works for npm 2.x
+      and 3.x in both the addon itself and projects that depend on this addon
+
+      This is an edge case b/c angular-material-source does not have a main
+      module we can require.resolve through node itself and similarily ember-cli
+      does not have such a hack for the same reason.
+
+      tl;dr - We want the non built scss files, and b/c this dep is only provided via
+      bower, we use this hack. Please change it if you read this and know a better way.
```

Comment:
Before I sign off for a bit should we leave the comment so we don't forget or remove it?